docs/copilot/prompt-crafting.md

```diff
@@ -25,7 +25,7 @@ Copilot works best when it has sufficient context to know what you're doing and
 
 #### Open files
```

Comment:
I initially missed the fact these were grouped into 2 paths (inline vs chat tips), so I had left a couple comments about "Should we divide these into 2 paths?" 😄 

But considering I missed the divide, I think readers could too. Maybe a quick note ~line 16 (so before getting into the inline path) could be helpful, something just noting that there are different ways to prompt craft for completions vs chat, and this doc has different tips for both. Maybe also linking to the headers (inline vs chat) so I don't have to scroll or ctrl+f to get to the chat tips if those are what I want.